Tella/Scenes/Reports/Outbox/OutboxDetailsView.swift

```diff
@@ -11,6 +11,8 @@ struct OutboxDetailsView<T: ServerProtocol>: View {
     @EnvironmentObject var mainAppModel : MainAppModel
     @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
     @EnvironmentObject private var sheetManager: SheetManager
+    @State private var isButtonDisabled = false
+    private let debounceInterval: TimeInterval = 1
```

Comment:
@Rimktarii @dhekra-rouatbi I added a 1s delay interval to prevent the crashes. I tested it tapping repeatedly and it's working fine. We could use this for now, let me know what do you think!